models/repo.go

```diff
@@ -654,7 +654,17 @@ func MigrateRepository(u *User, opts MigrateRepoOptions) (*Repository, error) {
 		return repo, UpdateRepository(repo, false)
 	}
 
-	if err = createUpdateHook(repoPath); err != nil {
+	repo, err = FinishMigrateRepository(repo, repoPath)
+	if err != nil {
+		return repo, err
+	}
+
+	return repo, UpdateRepository(repo, false)
+}
+
+// Finish migrating repository with things that don't need to be done for mirrors.
+func FinishMigrateRepository(repo *Repository, repoPath string) (*Repository, error) {
```

Comment:
Oh sorry. I think `CleanUpRepoInfo` would be better. It is not only used for mirrors.